flexget/plugins/filter/crossmatch.py

```diff
@@ -66,7 +69,7 @@ def on_task_filter(self, task, config):
         for entry in task.entries:
             for generated_entry in match_entries:
                 log.trace('checking if %s matches %s' % (entry['title'], generated_entry['title']))
-                common = self.entry_intersects(entry, generated_entry, fields)
+                common = self.entry_intersects(entry, generated_entry, fields, config.get('exact', True))
```

Comment:
I updated the default for exact in entry_intersects to True.  However it still needs the default supplied in config.get() or it returns None instead of True and doesn't take the default from entry_intersects